instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
> you typically create an account for you using the createsuperuser management command. This account should have full access.

@smarnach Yeah but [has_perm already checks for is_superuser](https://github.com/django/django/blob/9935f97cd203bdcc722bc3d4e96858e221d96ff8/django/contrib/auth/models.py#L261-L272) so you don't have to check for that explicitly.